### ISSUE TITLE
Undocumented m_img parameter

### ISSUE THREAD

**Issue by AGSPhoenix** [4chan/4chan-API#63]
I'm seeing an `m_img` parameter in threads. What is this?

Example thread: https://a.4cdn.org/k/thread/40218314.json

```
{"no":40218469,"now":"01\/07\/19(Mon)19:44:54","name":"Anonymous","com":"Look at my tactical shotgun<br><br><a href=\"#p40218314\" class=\"quotelink\">&gt;&gt;40218314<\/a><br>It depends","filename":"0586D749-EB4B-405F-B525-F7CF7B184C80","ext":".jpg","w":1936,"h":2592,"tn_w":93,"tn_h":125,"tim":1546908294712,"time":1546908294,"md5":"sWA3f9BnHTGl1vuQnveVpQ==","fsize":1323626,"resto":40218314,"m_img":1}
```

**Comment by bakugo** [comment]
4chan serves smaller copies of images (limited to 1024x1024) for mobile users. These images are in the same location as usual but the filename ends with `m`. This parameter indicates that this smaller image is available.

Example:
https://i.4cdn.org/k/1546907428891.jpg
https://i.4cdn.org/k/1546907428891m.jpg

**Comment by anjum-ahmed** [comment]
@bakugo Sauce on your profile pic?

**Comment by AGSPhoenix** [comment]
Fixed in #69.
